diode-server/proto/reconciler/v1/reconciler.proto

```diff
@@ -36,8 +36,38 @@ message RetrieveIngestionDataSourcesResponse {
   repeated IngestionDataSource ingestion_data_sources = 1;
 }
 
+// An ingestion log
+message IngestionLog {
+  string id = 1; // Unique identifier for the log
+  string state = 2; // State of the log
+  string data_type = 3; // Data type of the log
+  string request_id = 4; // Request ID associated with the log
+  int64 ingestion_ts = 5; // Timestamp of the ingestion
+  string value = 6; // The stored value in the JSON key
+}
+
+// The request to retrieve ingestion logs
+message RetrieveIngestionLogsRequest {
+  int32 number_of_entities = 1; // Override the default page size
```

Comment:
```suggestion
  int32 page_size = 1; // Override the default page size
```

Assuming default page size to be defined in the SDK/client and if not present there, we'd also set default page size on the server side and the `page_size` in the request to allow user to override SDK default.